src/app/pages/swap/components/swap-selected-asset-to.tsx

```diff
@@ -14,21 +14,25 @@ export function SwapSelectedAssetTo({ onChooseAsset, title }: SwapSelectedAssetT
   const [amountField] = useField('swapAmountTo');
   const [assetField] = useField('swapAssetTo');
 
-  const amountAsFiat = useAmountAsFiat(amountField.value, assetField.value.balance);
+  const amountAsFiat = useAlexSdkAmountAsFiat(
+    assetField.value?.balance,
+    assetField.value?.price,
+    amountField.value
```

Comment:
Any reason we can't default to 0 here, as if this is undefined, it means the user hasn't entered anything here? This prevents having to have this function use optional args.

Reply:
Yeah, bc we don't want to show zero right away in the input field. I think this needs to show the placeholder state when the asset is selected until the user enters a value? I'll double check the designs.